system/lib/wasmfs/file.h

```diff
@@ -93,17 +93,20 @@ class File : public std::enable_shared_from_this<File> {
 
 protected:
   File(FileKind kind, mode_t mode, backend_t backend)
-    : kind(kind), mode(mode), backend(backend) {}
+    : kind(kind), mode(mode), backend(backend) {
+    atime = mtime = ctime = time(NULL);
```

Comment:
This is a little worrying... we'll need to call into JS on each File creation. I wonder if maybe we can set it to 0 and only call JS if the time is actually needed, somehow..?

Reply:
I don't think this is so bad. File creation should be relatively rare and on nontrivial backends will require a bunch of work anyhow. If we want to have a correct implementation of these times, I don't think there's any way around doing this here.

In the future when we have virtual backends, it would be bad to set the times for the virtual files as well, but we can cross that bridge when we get to it.

Reply:
I guess this is hard to avoid. It may be a regression compared to the old FS though. In particular it will force inclusion of JS to handle time code, not just slow things down. But we can look into that later.